### PR TITLE
use Vue CLI for build

### PR DESCRIPTION
**Pull request by rangermeier**
I had some problems executing the current build infrastructure so I went ahead and replaced it with that by Vue CLI. As a result the name of the exported component changed to `VuePureLightbox` and also the path to the files in `/dist`. I hope these changes are acceptable.

Changes:

- remove custom webpack/browserify build steps in favor of vue-cli
- export as VuePureLightbox
- use LightboxDefaultLoader by default for loading slot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,30 +60,27 @@ Vue.use(Lightbox)
...
+<VuePureLightbox
   thumbnail="https://via.placeholder.com/350x150"
   :images="[
     { link: 'https://placekitten.com/1080/910', alt: 'Cat 1' },
     { link: 'https://placekitten.com/1080/920', alt: 'Cat 2' },
   ]"
 >
-  <lightbox-default-loader slot="loader"></lightbox-default-loader>
-  <div slot="content" slot-scope="{ url: { link, alt } }">
+  <div slot:loader>Loading…</div>
```
[the remark this answers is not in the document]
Comment (rangermeier) [reply README.md]: Looks like I couldn't decide wether to use the deprecated slot syntax or the unified syntax introduced in 2.6

[the remark this answers is not in the document]
Comment (DCzajkowski) [reply README.md]: And you used a mix of both? haha, that's funny

**Comment by DCzajkowski** [comment]
I will make a release of a major version just after I test everything is working as expected. Thanks again 😉

**Comment by rangermeier** [comment]
Thanks! I have to admit that I didn't test very thoroughly (e.g. using the built packages in older browsers, etc.). Also I have no experience so far with releasing vue packages. So yes please, test some more.

**Comment by DCzajkowski** [comment]
I have tested this and everything looks fine. There are some breaking changes, that's why I released it under 3.0.0 badge. I have also updated the codepen example and the readme instruction. I will look at the other PR other time, if you don't mind.

**Pull request merged**